### PR TITLE
Release 4/20 - Webtask.io Migrating to Node 8

### PR DESCRIPTION
**Pull request by NotMyself**
TODO:
- [x] Write Post
- [x] Schedule Publication with @RamiroND 
- [x] Request art assets
- [x] Request content review
- [x] Revise
- [ ] Ensure post is copied to new blog

screencapture-localhost-4000-blog-webtask-dot-io-migrating-to-node-8-2018-04-20-08_08_52

### CODE REVIEW

File: _posts/2018-04-19-webtask-dot-io-migrating-to-node-8.markdown [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+**Note:** None of the actions available in the import tool are destructive. Your Node 4 webtasks will be left intact until you choose to complete the migration by clicking the **Permanently switch to Node 8** button. Completing the migration will cause all webtasks on the Node 4 environment to be permanently deleted.
+
+For detailed instructions on the migration process, please review the [Webtask.io Migration Guide](https://github.com/auth0/wt-cli/wiki/Node8-webtask.io).
+
+
+## WT-CLI Migration Tools.
+
+After updating to the latest version of the wt-cli, any command a user issues using a Node 4 based profile will produce a warning about the impending end of life for Node 4. We have added a whole set of migration commands that will perform the migration process.
+
+- Update to the latest version by executing `npm update -g wt-cli`
```
Comment (yavorg) [inline _posts/2018-04-19-webtask-dot-io-migrating-to-node-8.markdown]: "version of the CLI"

File: _posts/2018-04-19-webtask-dot-io-migrating-to-node-8.markdown [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+**Note:** None of the actions available in the import tool are destructive. Your Node 4 webtasks will be left intact until you choose to complete the migration by clicking the **Permanently switch to Node 8** button. Completing the migration will cause all webtasks on the Node 4 environment to be permanently deleted.
+
+For detailed instructions on the migration process, please review the [Webtask.io Migration Guide](https://github.com/auth0/wt-cli/wiki/Node8-webtask.io).
+
+
+## WT-CLI Migration Tools.
+
+After updating to the latest version of the wt-cli, any command a user issues using a Node 4 based profile will produce a warning about the impending end of life for Node 4. We have added a whole set of migration commands that will perform the migration process.
+
```
Comment (yavorg) [inline _posts/2018-04-19-webtask-dot-io-migrating-to-node-8.markdown]: Maybe add a sentence here before the bullet list to the effect of: here is how to migrate using the CLI

File: _posts/2018-04-19-webtask-dot-io-migrating-to-node-8.markdown [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- node 8
+related:
+- 2018-04-18-slash-webtasks-migrating-to-node-8
+- 2016-09-14-build-a-serverless-slack-bot-with-webtask
+- 2016-06-28-building-serverless-apps-with-webtask
+---
+
+Today we've shipped Node 8 support for [Webtask.io](https://webtask.io)! Webtask.io is our free serverless sandbox that allows you to build applications without thinking about infrastructure. Over the last five years, Webtask.io has become very popular due to its ease of use and focus on quick HTTP response times. To date, 20,000+ developers have built their project using Webtask.io.
+
+Luckily, we have also released some new tools to help make the migration process a snap. I will briefly describe them in this post and point you to other sources of detailed information.
```
Comment (yavorg) [inline _posts/2018-04-19-webtask-dot-io-migrating-to-node-8.markdown]: Perhaps call out somewhere near the top that action is required before May 15th and to read on for details. Right now I have to get all the way to the bottom to know that if I do nothing, my stuff will get deleted.

Comment (RamiroND) [reply _posts/2018-04-19-webtask-dot-io-migrating-to-node-8.markdown]: You can use a TL;DR with something like "Webtask.io is migrating to Node 8 on **May 15, 2018**! Learn how to easily migrate your projects with our new tools"

Comment (NotMyself) [reply _posts/2018-04-19-webtask-dot-io-migrating-to-node-8.markdown]: That is a great idea @RamiroND! Thanks.

**Pull request merged**